files/SZWEEXEC/ZWEGEN00

```diff
@@ -0,0 +1,890 @@
+  x = DeleteDataSet(jclCopy'(ZWEIACFZ)')
+  x = DeleteDataSet(jclCopy'(ZWEITSS)')
+  x = DeleteDataSet(jclCopy'(ZWEITSSZ)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRA)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRT)')
+end
+if COMPARE('RTSS', CVTRAC_VAL) = 0 then do
+  x = DeleteDataSet(jclCopy'(ZWEIKRA1)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRA2)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRA3)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR1)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR2)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR3)')
+  x = DeleteDataSet(jclCopy'(ZWEIACF)')
+  x = DeleteDataSet(jclCopy'(ZWEIACFZ)')
+  x = DeleteDataSet(jclCopy'(ZWEIRAC)')
+  x = DeleteDataSet(jclCopy'(ZWEIRACZ)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRA)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRR)')
+end
+if COMPARE('ACF2', CVTRAC_VAL) = 0 then do
+  x = DeleteDataSet(jclCopy'(ZWEIKRT1)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRT2)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRT3)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR1)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR2)')
+  x = DeleteDataSet(jclCopy'(ZWEIKRR3)')      
+  x = DeleteDataSet(jclCopy'(ZWEIRAC)')
+  x = DeleteDataSet(jclCopy'(ZWEIRACZ)')
+  x = DeleteDataSet(jclCopy'(ZWEITSS)')
+  x = DeleteDataSet(jclCopy'(ZWEITSSZ)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRT)')
+  x = DeleteDataSet(jclCopy'(ZWENOKRR)')
+end
+
+say jcl' has been copied to 'jclCopy'.'
+
+/*
+================================================================================
+    Get a list of all members in the data set.
+================================================================================
+*/
+
+say 'Recording a list of members in 'jclCopy'.'
+
+call outtrap 'output.','*'
+address tso 'listd '"'"jclCopy"'"' members'
+call outtrap 'off'
+
+members. = 0
+
+do i = output.0 to 1 by -1
+  output.i = strip(output.i)
+  if COMPARE(output.i, '--MEMBERS--') = 0 then do
+    leave
+  end
+  call Print 'member - 'output.i
+  j = members.0 + 1
+  members.j.name = output.i
+  members.j.substitutions.0 = 0
+  members.0 = j
+end
+
+say 'The 'output.0' members in 'jclCopy' have been recorded.'
+
+/*
+================================================================================
+    Read each member record by record and store the substitutions required
+    for use later when the edit macro is invoked.
+================================================================================
+*/
+
+say 'Finding the substitutions in each member.'
+
+do i = 1 to members.0
+  if AllocateDataSet(jclCopy'('members.i.name')', 'zwejcl') > 0 then do
+    ExitWithRC(8)
+  end
+  if ReadFromDataSet('zwejcl') > 0 then do
+    ExitWithRC(8)
+  end
+  if FreeByDD('zwejcl') > 0 then do
+    ExitWithRC(8)
+  end
+  do j = 1 to !contentToRead.0
+    firstChar = POS('{', !contentToRead.j) + 1
+    do while firstChar > 1
+      lastChar = POS('}', !contentToRead.j, firstChar) - 1
+      len = lastChar - firstChar + 1
+      if len > 0 then do
+        sub = SUBSTR(!contentToRead.j, firstChar, len)
```

Comment:
We have to ensure `sub` has the same length as stored in `len`, otherwise:
Template: ` {   zowe.abc   }`
`sub="   zowe.abc   "`

L317 failing:
```
   317 +++     new = value('CFG.'members.i.substitutions.j)           
IRX0040I Error running ZWEGEN00, line 317: Incorrect call to routine  
ISPD117                                                               
The initially invoked CLIST ended with a return code = 20040          
```